### ISSUE TITLE
bib variables with '.' in name

### ISSUE THREAD

**Issue by courtney-bryce-hilton** [ropensci/bib2df#54]
Hi, thanks for the great package. 

I have found an edge-case that causes an issue with writing valid .bib files with df2bib. Specifically, I had a .bib entry that had 2 DOI values, like the following:

@article{test2022,
	doi = {DOI_PLACEHOLDER},
	doi = {DOI_PLACEHOLDER2}
}

parsing this with bib2df, and then rewriting it with df2bib, then results in something like:

@article{test2022,
	doi = {DOI_PLACEHOLDER},
	doi.1 = {DOI_PLACEHOLDER2}
}

However, '.' is not a valid character for a variable name for .bib files (I think; at least had issues knitting a .rmd). 

I suspect this would be an easy fix, where you could substitute '.' in variable names with '_' or something?

Cheers

**Comment by HedvigS** [comment]
On second though, I'm not sure this is really a bug. You shouldn't have duplicate fields in entries to start with, so the fact that something is breaking here may actually be a good thing as it alerts you to it.

You should be getting this warning:
```

Some BibTeX entries may have been dropped.
            The result could be malformed.
            Review the .bib file and make sure every single entry starts
            with a '@' 
```

What do you think @giabaio ?

**Comment by kellijohnson-NOAA** [comment]
I agree with @HedvigS that this should generate a warning/stop that duplicated fields are present. I saw this topic on [stack exchange](https://tex.stackexchange.com/questions/77653/multiple-doi-url-and-eprint-entries-in-biblatex) regarding how to deal with multiple urls and dois for a single entry that might be of interest for you. If we need more warning/error messages for this please feel free to reopen this issue and I will try to add those for you.
